CODEOWNERS

```diff
@@ -1,4 +1,4 @@
 # This file is described here:  https://help.github.com/en/articles/about-code-owners
 
-# Global Owners: These are brigadecore org maintainers + maintainers of this repo
-* @brigadecore/maintainers @brigadecore/docs-hindi-maintainers
+# Global Owners: These are Brigadecore org maintainers + maintainers of this repo
+* @Brigadecore/maintainers @Brigadecore/docs-hindi-maintainers
```

Comment:
Can you please reset this file?  https://stackoverflow.com/questions/37972753/git-reset-single-file-in-feature-branch-to-be-the-same-as-in-master

Reply:
pathspec 'origin/master' did not match any file(s) known to git

I am getting this error. Can you please help me out?

Reply:
I can even make these changes and submit a pull request again or maybe a new pull request containing only those changes.

Reply:
Took it offline and fixed.